smart-contracts/wasm-transform/src/artifact.rs

```diff
@@ -426,7 +445,7 @@ impl<'a, ImportFunc> From<BorrowedArtifact<'a, ImportFunc>> for Arc<OwnedArtifac
     fn from(a: BorrowedArtifact<'a, ImportFunc>) -> Self { Arc::new(a.into()) }
 }
 
-/// Internal opcode. This is mostly the same as OpCode, but with control
+/// Internal opcode. This is mostly the same as [`OpCode`], but with control
 /// instructions resolved to jumps in the instruction sequence, and function
 /// calls processed.
 #[repr(u8)]
```

Comment:
The variants below are not documented, but I guess that is fine as they are mostly selfexplanatory. Perhaps we can leave a link to the wasm documentation for reference.

Reply:
Yeah. There is a link to the wasm spec at the top-level. I have now added a link to the instructions page as well.